ca/django_ca/migrations/0012_auto_20181208_2105.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.1.4 on 2018-12-08 21:05
+
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.db import migrations
+from django_ca import ca_settings
+from django_ca.utils import write_private_file
+import os
+#import stat
+
+
+def migrate_data(apps, schema_editor):
+    CertificateAuthority = apps.get_model("django_ca", "CertificateAuthority")
+
+    # Check if current CA_DIR contains BASE_DIR
+    if os.path.isabs(ca_settings.CA_DIR):
+        if os.path.commonpath([ca_settings.CA_DIR, settings.BASE_DIR]) == settings.BASE_DIR:
+            prefix = os.path.relpath(ca_settings.CA_DIR, settings.BASE_DIR)
+    else:
+        prefix = ca_settings.CA_DIR
+
+    # Move keys from CAs
+    if CertificateAuthority.objects.count() > 0:
```

Comment:
```suggestion
    if CertificateAuthority.objects.count() > 0:
```

In general, this check is unnecessary in Django and relational databases. Simply loop over `CertificateAuthority.objects....`, and the loop will have zero iterations if there are no certificates. (this saves a database query!)